### PR TITLE
change format tally on list proposals

### LINKED ISSUE #2331: Show percentage instead of voting power on proposal list
**Is your feature request related to a problem? Please describe.**

The proposal list, [https://beta.cosmoswallet.io/#/governance/proposals](https://beta.cosmoswallet.io/#/governance/proposals) shows each proposal tally based on voting tally instead of percentage. 

This is confusing for new users as it's not clear what's the total distribution of votes on each of the options.

**Describe the solution you'd like**

Show percentage instead of voting power for the proposal tally in the LiProposal

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #2331 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: test/unit/specs/components/governance/__snapshots__/PageProposal.spec.js.snap [hunk 1]
```diff
@@ -1,6 +1,213 @@
...
+      <dl
+        class="info_dl colored_dl"
+      >
+        <dt>
+          Proposal Status
+        </dt>
+         
+        <dd>
+          
+            Voting started in 2 days.
```
Comment (jbibla) [inline test/unit/specs/components/governance/__snapshots__/PageProposal.spec.js.snap]: lol

File: PENDING.md [hunk 2]
```diff
@@ -1 +0,0 @@
-[Changed] [\#2287](https://github.com/cosmos/voyager/pull/2287) Use tags instead of commit hash @sabau
```
Comment (sabau) [inline PENDING.md]: nuu my pending is gone 😢

File: app/src/renderer/components/governance/LiProposal.vue [hunk 3]
```diff
@@ -59,15 +60,46 @@ export default {
...
       }
     },
+    totalVotes({ tally: { yes, no, no_with_veto, abstain } } = this) {
+      return BigNumber(yes)
+        .plus(no)
+        .plus(no_with_veto)
+        .plus(abstain)
+        .toNumber()
+    },
+    yesPercentage({ tally, totalVotes } = this) {
```
Comment (sabau) [inline app/src/renderer/components/governance/LiProposal.vue]: All those %percentage functions seems to create a bit of code duplication

File: changes/fedekunze_2331-percent-votes [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[Changed] [#2287](https://github.com/cosmos/lunie/pull/2287) Use tags instead of commit hash @sabau
```
Comment (faboweb) [inline changes/fedekunze_2331-percent-votes]: moved here @sabau

File: app/src/renderer/components/governance/LiProposal.vue [hunk 5]
```diff
@@ -31,23 +31,24 @@
     </td>
     <td>{{ `#` + proposal.proposal_id }}</td>
     <td class="li-proposal__value yes">
-      {{ tally.yes || `--` }}
+      {{ yesPercentage || `--` }}
```
Comment (sabau) [inline app/src/renderer/components/governance/LiProposal.vue]: I haven't double checked with data, but codewise seems the percentage functions already covers the `--`

File: changes/fedekunze_2331-percent-votes (hunk 4 above)
Comment (sabau) [inline changes/fedekunze_2331-percent-votes]: ah here it is XD will the new changelog tool work with two lines @faboweb ?

File: PENDING.md (hunk 2 above)
Comment (fedekunze) [reply PENDING.md]: added to changes haha

File: changes/fedekunze_2331-percent-votes (hunk 4 above)
Comment (faboweb) [reply changes/fedekunze_2331-percent-votes]: yes :)

**Pull request merged**